documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -20,6 +20,20 @@
               description: 'Prometheus %(prometheusName)s has failed to reload its configuration.' % $._config,
             },
           },
+          {
+            alert: 'PrometheusSDRefreshFailure',
+            expr: |||
+              rate(prometheus_sd_refresh_failures_total{%(prometheusSelector)s}[5m]) > 0
+            ||| % $._config,
+            'for': '10m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'Failed Prometheus SD refresh.',
+              description: 'Prometheus %(prometheusName)s has failed to refresh sd with mechanism {{$labels.mechanism}}' % $._config,
```

Comment:
Let's keep punctuation and capitalization consistent, i.e. "sd" → "SD" and end the sentence with a period.